swagger.yaml

```diff
@@ -4270,6 +4270,8 @@ definitions:
   disk:
     type: object
     properties:
+      id:
```

Comment:
Please add a swagger `description:` to:
`by_path`
`by_id`
`id`

so we understand what exactly is the difference between all of them

Reply:
done